Base_of_credit_card_vendors.txt

```diff
@@ -0,0 +1,2 @@
+Visa 13/16 4 
+MasterCard 16 2221-2720/51-55 
```

Comment:
Move the config class with vendor names masks and lengths params 
to proper place in structure depending on usage
If it's final config: src/main/resources or to centralized config service if it's used in company

If you want to test if you can read it then use : src/test/resources 

Consider improving format of this file - what if you want to have space in Vendor name (maybe JSON)